volplugin/handlers.go

```diff
@@ -168,6 +173,9 @@ func mount(master, host string) func(http.ResponseWriter, *http.Request) {
 			return
 		}
 
+		stopChan := addStopChan(uc.Request.Name)
+		go heartbeatMount(master, ttl, ut, stopChan)
```

Comment:
We should report a mount  and start the ttl thread only after `driver.Mount` call below succeeds, right?